tests/baselines/reference/jsFileCompilationExportAssignmentSyntax.errors.txt

```diff
@@ -1,9 +1,12 @@
 error TS5055: Cannot write file 'tests/cases/compiler/a.js' because it would overwrite input file.
+tests/cases/compiler/a.js(1,1): error TS1148: Cannot compile modules unless the '--module' flag is provided.
 tests/cases/compiler/a.js(1,1): error TS8003: 'export=' can only be used in a .ts file.
 
 
 !!! error TS5055: Cannot write file 'tests/cases/compiler/a.js' because it would overwrite input file.
-==== tests/cases/compiler/a.js (1 errors) ====
+==== tests/cases/compiler/a.js (2 errors) ====
     export = b;
     ~~~~~~~~~~~
+!!! error TS1148: Cannot compile modules unless the '--module' flag is provided.

```

Comment:
I agree this should be error only if !noEmit But what if we are emitting? what will we emit?
What should happen for 
```javascript
export class c {
}
```